### PR TITLE
Add kibana infra

### PR DESCRIPTION
**Pull request by gurevichdmitry**
From now, helm charts automation infra will deploy also kibana.
To reveal kibana on local host execute the following command:

`kubectl port-forward svc/cloudbeat-test-kibana -n kube-system 5601`

and then in browser navigate to `localhost:5601`

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -17,7 +17,9 @@ setup-env: install-kind create-kind-cluster
 create-vanilla-deployment-file:
    kustomize build {{kustomizeVanillaOverlay}} --output deploy/k8s/cloudbeat-ds.yaml
 
-build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image deploy-cloudbeat
+build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image
```
Comment (yashtewari) [inline justfile:20]: Why is the `deploy-cloudbeat` step removed?

File: tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-ds.yml [hunk 2]
```diff
@@ -5,15 +5,18 @@ metadata:
   name: cloudbeat
   namespace: kube-system
   labels:
+    catf: related
```
Comment (yashtewari) [inline tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-ds.yml:8]: What's this for?

**Review by yashtewari — APPROVED** [review]
LGTM

File: justfile (hunk 1 above)
Comment (gurevichdmitry) [reply justfile]: Fixed

File: tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-ds.yml (hunk 2 above)
Comment (gurevichdmitry) [reply tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-ds.yml:8]: These labels are used to identify deployments related to catf (cloudbeat automated tests framework)

**Comment by DaveSys911** [comment]
@yashtewari @gurevichdmitry Perhaps we can consider backporting this, so we won't run into pesky conflicts with the 8.5 branch.

**Comment by gurevichdmitry** [comment]
> @yashtewari @gurevichdmitry Perhaps we can consider backporting this, so we won't run into pesky conflicts with the 8.5 branch.

As it was agreed with @kfirpeled no need to backport tests and infra to branched out versions.

**Pull request closed without merging**